app/models/concerns/delivery_area.rb

```diff
@@ -0,0 +1,37 @@
+module Groups
```

Comment:
file name suggests delivery area but this is a permissions module, make sure your file names and module names match or are atleast related